modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/util/ObjectFieldBusinessTypeUtil.java

```diff
@@ -37,6 +37,8 @@ public static List<Map<String, String>> getObjectFieldBusinessTypeMaps(
 					objectFieldBusinessType.getDescription(locale)
 				).put(
 					"label", objectFieldBusinessType.getLabel(locale)
+				).put(
+					"value", objectFieldBusinessType.getName()
```

Comment:
@danvb112 Following the existing pattern, you should call this property "name", based on the name of the method returning it.

Reply:
Hey @naraandrade, Picker requires a key called `value` in the JSON that we are sending to it, so I need this key, what I could do is name this key here as `name`, and in the frontend take this json and put the key `value` there with the same value, what do you think?

Reply:
> name this key here as name, and in the frontend take this json and put the key value there with the same value
It's better to follow that approach. We should keep the backend patterns.

Reply:
Thanks for the suggestion @naraandrade ! Done!